roles/wisdom_warmup_model/vars/main/resources.yml

```diff
@@ -0,0 +1,6 @@
+---
+wisdom_namespace: wisdom
+wisdom_warmup_entrypoint_template: 'templates/wisdom-warmup_entrypoint.yml.j2'
+wisdom_warmup_pod: 'roles/wisdom_warmup_model/files/wisdom-warmup_pod.yml'
+warmup_concurrency: '16'
+warmup_total_requests: '4096'
```

Comment:
you should move this to the Python entrypoint, so that you can easily configure it

overall, I think the `/vars` directory should be only for ... strict constants like role file paths, and `/config` for everything else that you might want to change one day